lib/package.gi

```diff
@@ -1936,6 +1952,7 @@ InstallGlobalFunction( ExtendRootDirectories, function( rootpaths )
     local i;
 
     rootpaths:= Filtered( rootpaths, path -> not path in GAPInfo.RootPaths );
+    # TODO: validate the paths???
```

Comment:
remove? or is there a plan to add something?

Reply:
removed. this was an artifact from #933